recipe/meta.yaml

```
@@ -1,30 +1,25 @@
 {% set name = "psutil" %}
-{% set version = "5.9.0" %}
-{% set sha256 = "869842dbd66bb80c3217158e629d6fceaecc3a3166d3d1faee515b05dd26ca25" %}
+{% set version = "7.0.0" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7be9c3eba38beccb6495ea33afd982a44074b78f28c434a1f51cc07fd315c456
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install --ignore-installed --no-deps .
-  # we currently do not support python versions earlier than 3.7
-  skip: true  # [py<37]
+  number: 0
+  skip: true  # [py<36]
 
 requirements:
   build:
     - {{ compiler('c') }}
```

```suggestion
    - {{ stdlib('c') }}
    - {{ compiler('c') }}
```
Should use stdlib anywhere you need a compiler to build an output.